src/reference/00-Getting-Started/07A-Task-Graph.md

```diff
@@ -0,0 +1,324 @@
+---
+out: Task-Graph.html
+---
+
+  [Basic-Def]: Basic-Def.html
+  [Scopes]: Scopes.html
+  [Make]: https://en.wikipedia.org/wiki/Make_(software)
+  [Ant]: http://ant.apache.org/
+  [Rake]: https://ruby.github.io/rake/
+
+Task graph
+----------
+
+This page explains `build.sbt` definition in more detail.
+It assumes you've read [.sbt build definition][Basic-Def] and
+[scopes][Scopes].
```

Comment:
I'm wondering if 'make' can be described as a singly-scoped SBT, and in turn, fully-scoped SBT as merely a "depth dimension" of makefiles? 

If things could be inexactly considered in those terms, the novice could put their mind at rest not to worry about scopes until later, but at least be mentally prepared for when they were presented.